saleor/graphql/product/types/products.py

```diff
@@ -499,6 +499,7 @@ class Meta:
             "seo_title",
             "updated_at",
             "weight",
+            "visible_in_listings",
```

Comment:
This field should be available to customers?

Reply:
I did it in the same way as `is_published` field. But maybe both of these fields should be protected be some permission.  What do you think @patrys ?

Reply:
We should leave this field as public. In some cases you want to hide some navigation depends on this field.